spec/integration/ai_chat_integration_spec.rb

```diff
@@ -0,0 +1,303 @@
+# frozen_string_literal: true
+
+require "spec_helper"
+
+RSpec.describe "AI::Chat Integration", :integration do
+  describe "basic chat functionality" do
+    it "generates a response to a simple question" do
+      chat = AI::Chat.new
+      chat.user("What is 2 + 2?")
+
+      response = chat.generate!
+
+      expect(response).to be_a(String)
+      expect(response).to match(/4|four/i)
+      expect(chat.messages.count).to eq(2)
+      expect(chat.messages.last[:role]).to eq("assistant")
+      expect(chat.messages.last[:response]).to be_a(AI::Response)
+    end
+
+    it "maintains conversation context across multiple turns" do
```

Comment:
This might be more clear:
```suggestion
    it "maintains conversation context across multiple generated messages" do
```